lib/bamboo/adapters/mandrill_helper.ex

```diff
@@ -36,6 +36,59 @@ defmodule Bamboo.MandrillHelper do
   end
 
   @doc """
+  Set merge_vars that are used by Mandrill 
+
+  A convenience function for: 
+
+      email
+      |> put_param(email, "merge_vars", [
+        %{
+          rcpt: "user1@example.com",
+          vars: [
+            %{
+              "name": "full_name",
+              "content": "User 1"
+            }
+          ]
+        },
+        %{
+          rcpt: "user2@example.com",
+          vars: [
+            %{
+              "name": "full_name",
+              "content": "User 2"
+            }
+          ]
+        }
+      ])
+
+  ## Example
```

Comment:
What do you think about moving the example above the "A convenience function for:"